README.md

```diff
@@ -8,7 +8,6 @@ Toolbox for the Modeling and Analysis of Thermodynamic Systems <br>
 For questions, comments, and general support see the <b> <a href= "https://groups.google.com/forum/#!forum/t-mats-user-group" >T-MATS user's forum</a></b>. <br>
 Please take a moment and fill out our <b> <a href= "https://docs.google.com/forms/d/1cjcCyOKZpV49-gsdGGaKHUsZ2c2fKX5mwkKrntt60Eo/viewform?usp=send_form" >Survey</a></b>, user interest and feedback allows us to continue working on this package.<br><br>
 
-<b>Introduction</b> <br>
```

Comment:
I wonder what this is.

Reply:
I can tell you

Reply:
Awsome.

Reply:
Is it a reference to this?

https://github.com/diffplug/T-MATS/blob/74a3671cfcb04bd76f9b113ac98ecc4a14b05970/Trunk/Install_TMATS.m#L17

Reply:
ned

Reply:
edgar